istioctl/pkg/install/k8sversion/version.go

```diff
@@ -29,7 +29,7 @@ import (
 const (
 	// MinK8SVersion is the minimum k8s version required to run this version of Istio
 	// https://istio.io/docs/setup/platform-setup/
-	MinK8SVersion = 16
+	MinK8SVersion = 17
```

Comment:
This is the line that we have been discussing, and would yield a message like:
```
The Kubernetes version v1.15.10 is not supported by Istio 1.9.0-beta.0. The minimum supported Kubernetes version is 1.17.
```
Based on the `support` statement being 1.17-1.20, I think this is probably appropriate. Those using Kubernetes 1.16 (which isn't qualified) would see this message. I guess the question if that is still appropriate could be brought to the TOC.

Reply:
Right, and this is used by `istioctl x precheck` to produce those messages, so before installing Istio on any Kubernetes cluster user will run the precheck command to see if the version of Istio that they are installing is compatible with k8.